src/api/app/controllers/source_controller.rb

```diff
@@ -1450,13 +1454,17 @@ def reparse_backend_package(spackage, sproject)
 
   # POST /source/<project>/<package>?cmd=release
   def package_command_release
-    pkg = Package.get_by_project_and_name params[:project], params[:package], use_source: true, follow_project_links: false
+    pkg = Package.get_by_project_and_name params[:project], params[:package], use_source: true, follow_project_links: false, follow_multibuild: true
```

Comment:
Please use speaking variable names so `pkg` => `package`

Reply:
we can do so, but this is IMHO outside of the scope of this commit since there are plenty places of this all over the code. It used to be our standard to avoid to long unreadable code. I don't mind to change it, but not only for a single line...

Reply:
I would argue that trying to make code more readable by shortening variable names actually makes the code less readable but ok fine for me if you don't want to change it in this PR. :+1: